api/src/main/java/com/velocitypowered/api/proxy/Player.java

```diff
@@ -148,9 +148,7 @@ public interface Player extends
   /**
    * Clears the tab list header and footer for the player.
    *
-   * @deprecated Use {@link TabList#clearHeaderAndFooter()}.
    */
-  @Deprecated
```

Comment:
Can we introduce a new method instead of un-deprecating this one? `clearPlayerListHeaderAndFooter`